configuration/wireless/wireless-cli.md

```diff
@@ -59,12 +59,12 @@ Reconfigure the interface with `wpa_cli -i wlan0 reconfigure`.
 
 You can verify whether it has successfully connected using `ifconfig wlan0`. If the `inet addr` field has an address beside it, the Raspberry Pi has connected to the network. If not, check that your password and ESSID are correct.  
 
-On the Raspberry Pi 3 Model B+ and above, you will also need to set the country code, so that the 5G networking can choose the correct frequency bands. You can either use the raspi-config application and select the localisation option, or edit the `wpa_supplicant.conf` file and add the following. (Note you need to replace 'GB' with the 2 letter ISO code of your country. See [Wikipedia](https://en.wikipedia.org/wiki/ISO_3166-1) for a list of 2 letter ISO 3166-1 country codes.)
+On the Raspberry Pi 3B+ and Raspberry Pi 4B, you will also need to set the country code, so that the 5GHz networking can choose the correct frequency bands. You can do this using the `raspi-config` application: select the 'Localisation Options' menu, then 'Change Wi-Fi Country'. Alternatively, you can edit the `wpa_supplicant.conf` file and add the following. (Note: you need to replace 'GB' with the 2 letter ISO code of your country. See [Wikipedia](https://en.wikipedia.org/wiki/ISO_3166-1) for a list of 2 letter ISO 3166-1 country codes.)
 ```
 country=GB
 ```
 
-Note that with the latest Buster Raspbian release, you must ensure that the `wpa_supplicant.conf` file contains the following information at the top.
+Note that with the latest Buster Raspbian release, you must ensure that the `wpa_supplicant.conf` file contains the following information at the top:
```

Comment:
IIRC the documentation _only_ targets the latest release of Raspbian (anything else is "unsupported"), so maybe this "latest Buster Raspbian" caveat should just be incorporated directly into the main text?

Reply:
Good point, although the docs do tend to call out differences for a time after they are implemented.

> @JamesH65 Is #109 still relevant? Or has the "documentation style guide" changed since then?

Wikipedia says Wi-Fi is the correct spelling, but according to that Liz in 2014 said that WiFi was correct.

Do you want me to excise 'Wi-Fi' and replace with 'wireless'?

Reply:
Turns out, upon closer inspection, that is not the style guide I'm looking for. The one I saw was more detailed; I suspect it has been moved behind closed doors.